MultiVI_tutorial.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>A paragraph above describing why you're doing this would be better than comments. Generally, using text cells is better than comments. Would be great to do this in all relevant cases</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>also, why not also add a pure RNA pbmc dataset from 10x? We have a function in scvi.data that gets anndata from like 25 different 10x datasets</p>